### ISSUE TITLE
Display the rule after an error in flycheck

### ISSUE THREAD

**Issue by mrcnski** [abingham/flycheck-vale#13]
When I see a `vale` error the name of the rule that was violated is not shown. This would be useful for when I want to disable the rule. I have to run vale from the command line to get the name of the rule. Other flycheck packages, such as shellcheck, show the rule in parentheses.

Example:

```
 scratch…   170 176 warning         'all of' is too wordy (vale)
```

**Comment by mrcnski** [comment]
Example from command line:

```
 170:176  warning  'all of' is too wordy           write-good.TooWordy  
```

**Comment by juergenhoetzel** [comment]
#15 fixes the issue
